fbw-a380x/src/systems/instruments/src/MFD/pages/ATCCOM/MfdAtccomDAtis.tsx

```diff
@@ -31,68 +32,44 @@ export class MfdAtccomDAtis extends DisplayComponent<MfdAtccomDAtisProps> {
           <DAtisBlock
```

Comment:
Shouldn't there be one for each airport, not just 2?

Also, won't this not update as airports are added/removed?

Look into `ArraySubject`.